lib/load.php

```diff
@@ -120,6 +120,11 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/global-styles.php';
 require __DIR__ . '/pwa.php';
 
+// TODO: Before this PR merges, move this to be a part of the style engine package.
+// Part of the build process should be to copy the PHP file to the correct location,
+// similar to the loading behaviour in `blocks.php`.
+require __DIR__ . '/class-wp-style-engine-gutenberg.php';
```

Comment:
If we move this to the style engine package, won't we need to create the same `compat/wordpress-x.y` folder structure in that package as we have in `lib`, to avoid clashes once the initial version of this is in Core?